codes/zig/chapter_heap/heap.zig

```diff
@@ -18,13 +18,13 @@ fn testPush(comptime T: type, mem_allocator: std.mem.Allocator, heap_push: anyty
     var heap = heap_push;
     try heap.add(val);  //元素入堆
     std.debug.print("\n元素 {} 入堆后\n", .{val});
-    try inc.PrintUtil.printHeap(T, mem_allocator, heap, true);
+    try inc.PrintUtil.printHeap(T, mem_allocator, heap);
 }
 
 fn testPop(comptime T: type, mem_allocator: std.mem.Allocator, heap_pop: anytype) !void {
```

Comment:
Replace `heap_pop` with `heap`. We should make the naming clear to understand.

Reply:
> Replace `heap_pop` with `heap`. We should make the naming clear to understand.

Done.